src/profile/v100/entity/abstract_entity_type.rs

```diff
@@ -19,37 +19,8 @@ use radix_engine_common::types::EntityType as ScryptoEntityType;
 )]
 #[repr(u32)] // it is u32 since used in Derivation Paths (CAP26) where each component is a u32.
 pub enum AbstractEntityType {
```

Comment:
So this is meant only to describe an Entity in the sense of either Account or Identity, but not other address kind?